templates/navigation-submenu.twig

```diff
@@ -0,0 +1,25 @@
+<nav class='nav-submenu'>
+	<div class='nav-submenu-wrapper'>
+		<ul>
+		{% for key,item in item.children %}
+			<li>
+				{% if fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == act_page_id %}
```

Comment:
I think we can remove these special cases already since we're dropping the Act/Explore navigation, and just use `item.title` everywhere

Reply:
Shouldn't they behave as a link? And what about Donate links?

Reply:
They will still behave as links 🤔 This is just for the Google Analytics action parameter, and if I understood correctly with the new IA we no longer treat Act and Explore as special pages so here we should use `item.title` everywhere... Anyway, I see that we still use this in the navigation bar template so I guess we can keep this change for later in the process

Reply:
Sorry I miss explained 😶 for links I referred to `data-*` values. I asked about these values a week ago to Julia, so, is this [answer](https://jira.greenpeace.org/browse/PLANET-6509?focusedCommentId=245076&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-245076) solve your doubt?

Reply:
Well not really, it looks like she agrees with me? Julia says to use `item.title`, she doesn't mention exceptions for the Act and Explore pages... But this is not related to the Donate button so it can be solved in another PR 🙃